src/components/ResizablePanels.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <div class="resizable-panels">
+    <div class="resizable-panels__panel" :style="leftPanelWidth">
+      <slot name="left-panel">Left panel</slot>
+    </div>
+
+    <div class="resizable-panels__resizer"
+      @mousedown="startResize()"
+    >
+      <div class="resizable-panels__line"></div>
+      <div class="resizable-panels__line"></div>
+      <div class="resizable-panels__line"></div>
+    </div>
+
+    <div class="resizable-panels__panel" :style="rightPanelWidth">
+      <slot name="right-panel">Right panel</slot>
+    </div>
+  </div>
+</template>
+<script  lang="ts">
+import Vue from 'vue'
+import { Component } from 'vue-property-decorator'
+
+@Component({
+  name: 'resizable-panels'
+})
+export default class ResizablePanels extends Vue {
+  private isResized: boolean = false;
+  private ratio: number = 0.4;
+
+  get leftPanelWidth() {
+    return {
+      width: `${this.ratio * 100}%`,
+    };
+  }
+
+  get rightPanelWidth() {
+    return {
+      width: `${1 - (this.ratio * 100)}%`,
+    };
+  }
+
+  startResize() {
+    const containerSize = this.$el.getBoundingClientRect();
+    const containerSizeOnAxis = containerSize.width;
+
+    const mousemoveListener = (e: any) => {
```

Comment:
`e` should be of type `MouseEvent`?
https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent